log-pipeline.yml

```diff
@@ -65,10 +76,33 @@ services:
 
   log-api:
     image: monasca/log-api:${MON_LOG_API_VERSION}
+    environment:
+      LOGSPOUT: ignore
+      LOGSTASH_FIELDS: "service=monasca-log-api"
     depends_on:
       - keystone
       - zookeeper
       - kafka
       - memcached
     ports:
       - "5607:5607"
+
+  log-agent:
+    image: monasca/log-agent:latest
+    environment:
+      LOGSPOUT: ignore
+      LOGSTASH_FIELDS: "service=monasca-log-agent"
```

Comment:
Is  `LOGSTASH_FIELDS` required if you ignore the `log-agent` from monitoring anyway ?

Reply:
I would prefer to have all services tagged and ignore them on demand, someone would like to stop ignoring so service so he remove one line and it's tagged properly.

Reply:
Well, we can always see that in this context. @timothyb89 you're fine with adding these fields to each service ?